src/dorsal.js

```diff
@@ -355,12 +359,11 @@ DorsalCore.prototype.unwire = function(el, pluginName) {
  */
 DorsalCore.prototype.wire = function(el, pluginName) {
     var deferred = new DorsalDeferred(this.ELEMENT_TO_PLUGINS_MAP),
-        pluginKeys = this.registeredPlugins(),
-        responses,
+        responses = [],
         action;
 
     if (!this.plugins) {
-        throw new Error('No plugins registered with Dorsal');
+        console.warn('No plugins registered with Dorsal');
```

Comment:
I think it's a feature that it's scraped out by the minifier, because it's just a message for developers. Same argument could be made about full-browser support, though on shakier ground, certainly.